### PR TITLE
BUG remove F90 and F77

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by phil-blain** [comment]
thanks @beckermr !

File: recipe/deactivate-gfortran.sh [hunk 1]
```diff
@@ -105,6 +105,8 @@ _tc_activation \
 # extra ones:
 unset FC
 unset F95
+unset F90
+unset F77
```
Comment (isuruf) [inline recipe/deactivate-gfortran.sh:109]: This is not the correct way. See https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/master/recipe/deactivate-gfortran.sh#L107-L112

Comment (beckermr) [reply recipe/deactivate-gfortran.sh:109]: This was copied out of the upstream recipe I think. Will fix.

**Pull request merged**